wagtail/admin/templates/wagtailadmin/workflows/edit.html

```diff
@@ -59,7 +58,14 @@
                                 <ul>
                                     {% if can_enable %}
                                         <li>
-                                            <button type="submit" class="button no" data-enable-action>{{ view.enable_item_label }}</button>
+                                            <button class="button no" 
+                                                type="submit"
+                                                data-controller="w-action"
+                                                data-w-action-csrf-token-value = '{{ csrf_token|escapejs }}'
+                                                data-w-action-key-name-value = 'csrfmiddlewaretoken'
```

Comment:
We should be able to import the csrf token from client config. 

See how it's done in https://github.com/wagtail/wagtail/blob/main/client/src/entrypoints/admin/preview-panel.js

Note. Import the config in the controller - no need to use csrf key / value as Stimulus values at all.